addons/binding/org.openhab.binding.airvisualnode/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,145 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="airvisualnode"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="avnode">
+		<label>AirVisual Node</label>
+		<description>AirVisual Node air quality monitor</description>
+
+		<!-- Channel groups -->
+
+		<channel-groups>
+			<channel-group id="measurements" typeId="Measurements" />
+			<channel-group id="status" typeId="Status" />
+		</channel-groups>
+
+		<!-- Configuration parameters -->
+
+		<config-description>
+			<!-- Required parameters -->
+			<parameter name="address" type="text" required="true">
+				<context>network_address</context>
+				<label>Node Network Address</label>
+				<description>Node network address</description>
+				<required>true</required>
+			</parameter>
+			<parameter name="username" type="text" required="true">
+				<label>Node Username</label>
+				<description>Node network username</description>
+				<default>airvisual</default>
+			</parameter>
+			<parameter name="password" type="text" required="true">
+				<context>password</context>
+				<label>Node Password</label>
+				<description>Node network password</description>
+			</parameter>
+			<!-- Advanced parameters -->
+			<parameter name="share" type="text">
+				<label>Share Name</label>
+				<description>Node network share name</description>
+				<default>airvisual</default>
+				<advanced>true</advanced>
+			</parameter>
+			<parameter name="refresh" type="integer" min="30" unit="s">
+				<label>Refresh Interval</label>
+				<description>Node data fetches interval (in seconds)</description>
+				<default>60</default>
+				<unitLabel>s</unitLabel>
+				<advanced>true</advanced>
+			</parameter>
+		</config-description>
+	</thing-type>
+
+	<!-- Channel group types -->
+
+	<channel-group-type id="Measurements">
+		<label>Measurements</label>
+		<channels>
+			<channel id="co2_ppm" typeId="Co2_ppm" />
+			<channel id="humidity" typeId="Humidity" />
+			<channel id="aqi_cn" typeId="Aqi_cn" />
+			<channel id="aqi_us" typeId="Aqi_us" />
+			<channel id="pm_25" typeId="Pm_25" />
+			<channel id="temp_celsius" typeId="Temp_celsius" />
+			<channel id="temp_fahrenheit" typeId="Temp_fahrenheit" />
+		</channels>
+	</channel-group-type>
+
+	<channel-group-type id="Status">
+		<label>Status</label>
+		<channels>
+			<channel id="battery_level" typeId="system.battery-level" />
+			<channel id="wifi_strength" typeId="system.signal-strength" />
```

Comment:
See my comment below, use a system channel for this.

Reply:
Hmm, could you clarify please, do you mean I should change these channel ids? Because both channels are already referenced to a system channel typeIds. Should I just rename channel ids to `battery-level` and `signal-strength`, or some more changes are required?

Reply:
Upps, sorry apparently I stopped reading after `<channel id="battery_level"`... Sorry my fault.